data/i18n/en/en.toml

```diff
@@ -70,6 +58,15 @@ other = "Official Kubernetes CVE List"
 [cve_url]
 other = "CVE URL"
 
+[cve_table_date_before]
+other = "(last updated: "
+
+[cve_table_date_format]
+other = "02 Jan 2006 15:04:05 MST"
+
+[cve_table_date_after]
+other = ")"
+
```

Comment:
The collation order is wrong here. In English, `t` collates before `u`.

Reply:
I'm sorry I really tried but I don't understand what you mean 😞, is there a problem with "last updated"?

Reply:
Try this order for the keys:

* `cve_id`
* `cve_issue_url`
* `cve_json_external_url`
* `cve_json_id`
* `cve_json_summary`
* `cve_json_url`
* `cve_summary`
* `cve_table`
* ~`cve_url`~
* `cve_table_date_before`
* `cve_table_date_format`
* `cve_table_date_after`
* **`cve_url`**

Reply:
aaaaah I did not even notice they were ordered alphabetically ok ok

Reply:
Ok I just removed `cve_url` because it was unused anyway